ha/alert/alert_monitor.py

```diff
@@ -62,32 +64,28 @@ def process_alert(self):
         """
         Process alert
         """
-
         self._validate_pacemaker()
         self._load_recipient()
 
         # validate event type and get class
         try:
-             event_type = self._validate_event()
-             # Redirect the alert to appropriate monitor for further processing
-             if event_type:
-                 Log.info(f"Handling the event: {str(self.crm_env)}")
-                 self._redirect_alert(event_type)
-             else:
-                 Log.info(f"Identified unknown event: {str(self.crm_env)}")
-
+            # Redirect the alert to appropriate monitor for further processing
+            if self.alert_event_module and self.alert_event_type:
+                Log.info(f"Handling the event: {str(self.crm_env)}")
+                self._redirect_alert()
+            else:
+                Log.info(f"Identified unknown event by HA: {str(self.crm_env)}")
```

Comment:
Can you please change the wording to something like "Identified unknown /unsupported event "

Reply:
done